project/MiMa.scala

```diff
@@ -17,10 +17,230 @@ object MiMa extends AutoPlugin {
+      ProblemFilters.exclude[MissingClassProblem]("akka.routing.SmallestMailboxRouter"),
+      // object akka.routing.ScatterGatherFirstCompletedRouter does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.routing.ScatterGatherFirstCompletedRouter$"),
+      // interface akka.routing.DeprecatedRouterConfig does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.routing.DeprecatedRouterConfig"),
+      // object akka.routing.RoundRobinRouter does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.routing.RoundRobinRouter$"),
+      // method nrOfInstances()Int in class akka.routing.FromConfig does not have a correspondent in new version
+      ProblemFilters.exclude[MissingMethodProblem]("akka.routing.FromConfig.nrOfInstances"),
+      // method toString()java.lang.String in object akka.routing.BalancingPool does not have a correspondent in new version
+      ProblemFilters.exclude[MissingMethodProblem]("akka.routing.BalancingPool.toString"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.remote.RemoteSettings.LogRemoteLifecycleEvents"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.cluster.Cluster.publishCurrentClusterState"),
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.InternalClusterAction$PublishCurrentClusterState$"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.cluster.ClusterSettings.AutoDown"),
+      // class akka.cluster.routing.ClusterRouterSettings does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.ClusterRouterSettings"),
+      // object akka.cluster.routing.ClusterRouterConfig does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.ClusterRouterConfig$"),
+      // method nrOfInstances()Int in class akka.cluster.routing.ClusterRouterPool does not have a correspondent in new version
+      ProblemFilters.exclude[MissingMethodProblem]("akka.cluster.routing.ClusterRouterPool.nrOfInstances"),
+      // object akka.cluster.routing.AdaptiveLoadBalancingRouter does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.AdaptiveLoadBalancingRouter$"),
+      // object akka.cluster.routing.ClusterRouterSettings does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.ClusterRouterSettings$"),
+      // class akka.cluster.routing.AdaptiveLoadBalancingRouter does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.AdaptiveLoadBalancingRouter"),
+      // class akka.cluster.routing.ClusterRouterConfig does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.ClusterRouterConfig"),
```

Comment:
Hmm, I guess we cannot automatically verify that the moved versions are identical, but we should keep this in mind.